android/assets/jsons/translations/Spanish.properties

```
@@ -6362,13 +6362,13 @@ In research agreements, you and another civilization decide to jointly research
 The amount of ⍾Science you receive at the end is dependent on the ⍾Science generated by your cities and the other civilization's cities during the agreement - the more, the better! = La suma de ⍾ciencia que recibes al final depende de la ⍾ciencia generada por tus ciudades y por las ciudades de la otra civilización durante el acuerdo.
 
  # Requires translation!
-Defensive Pacts = 
+Defensive Pacts = Pactos Defensivos
  # Requires translation!
-Defensive pacts allow you and another civ to protect one another from aggressors.\nOnce the defensive pact is signed, you will be drawn into their future defensive wars, just as they will be drawn into your future defensive wars. Declaring war on any Civ will remove all of your defensive pacts. You will have to re-sign them to use their effect. = 
+Defensive pacts allow you and another civ to protect one another from aggressors.\nOnce the defensive pact is signed, you will be drawn into their future defensive wars, just as they will be drawn into your future defensive wars. Declaring war on any Civ will remove all of your defensive pacts. You will have to re-sign them to use their effect. = Los pactos defensivos permiten que dos civilizaciones se protejan de un agresor.\nUna vez que se haya firmado el pacto defensivo, serás incluido en las futuras guerras defensivas del aliado, al igual que el aliado se unirá en tus futuras guerras defensivas.\nAl declarar la guerra contra otra Civ (guerra ofensiva) se anularán todos tus pactos defensivos. Por lo que tendrás que volver a firmarlos para usar su efecto.
  # Requires translation!
-Be cautious when signing defensive pacts because they can bring you into wars that you might not want to be in. = 
+Be cautious when signing defensive pacts because they can bring you into wars that you might not want to be in. Tenga cuidado al firmar pactos defensivos porque te pueden arrastrar a guerras que no te convengan.
```

@Caballero-Arepa Te estabas olvidando de un símbolo de igual. 
```suggestion
Be cautious when signing defensive pacts because they can bring you into wars that you might not want to be in. = Tenga cuidado al firmar pactos defensivos porque te pueden arrastrar a guerras que no te convengan.
```

---

Ay cierto. Gracias mano

---

> Ay cierto. Gracias mano
¡De nada!